R/quosure-get_code.R

```diff
@@ -2,16 +2,16 @@
 #'
 #' @name get_code
 #' @param object (`Quosure`)
+#' @param deparse (`logical(1)`) if the returned code should be converted to character.
```

Comment:
should this be handled on the caller side?

Reply:
That will then mean every module (and module developer) will need to change from:


```
...
verbatim_content = reactive(teal.code::get_code(output_q())),
...

card$append_src(paste(teal.code::get_code(output_q()), collapse = "\n"))
```

to adding extra deparse statements - which I'm not sure is what we want?

Reply:
We can remove `deparse` and limit `get_code` to always return character. I don't see the sense of returning "raw" `@code` (expression).